infra/util/BoundedString.hpp

```diff
@@ -430,7 +431,7 @@ namespace infra
 
     template<class T>
     BoundedStringBase<T>::BoundedStringBase(T* s)
-        : range(s, s + std::strlen(s))  //NOSONAR
+        : range(s, s + std::strlen(s)) // NOSONAR
```

Comment:
Does Sonar understand // NOSONAR with the extra space?

Reply:
Since we are not getting a finding on that line; apparently yes.